alpine-base/root/etc/fix-attrs.d/01-resolver-resolv

```diff
@@ -0,0 +1 @@
+/etc/resolv.conf false doesntexist,0:1000 0664 0664
```

Comment:
It's just a user that doesn't exist. See https://github.com/just-containers/s6-overlay#fixing-ownership--permissions

> account: Target account. It's possible to default to fallback uid:gid if the account isn't found. For example, nobody,32768:32768 would try to use the nobody account first, then fallback to uid 32768 instead.

So I just needed it to fallback :)